airbyte-config/config-models/src/main/resources/type_exploration/airbyte.proto

```diff
@@ -0,0 +1,76 @@
+import "google/protobuf/timestamp.proto";
+
+// =============================== DEFINITIONS ================================
+// types supported by airbyte.
+enum FieldType {
+  STRING = 0;
+  INT = 1;
+  LONG = 2;
+  BOOLEAN = 3;
```

Comment:
likely `DOUBLE` or `FLOAT`

Reply:
for sure. all of this is shorthand for now. trying to make sure i pick out examples that'll test the edges.